engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -771,7 +771,8 @@ object ProgramExecutionSupport {
               p.getLocation().getEncapsulatingSourceSection() match {
                 case ss: SourceSection =>
                   logger.warn(
-                    s"Error at ${ss.getCharIndex()}-${ss.getCharEndIndex()} (e.g. `${ss
+                    s"Error at ${ss.getCharIndex()}-${ss
+                      .getCharEndIndex()} in ${ss.getSource.getPath} (e.g. `${ss
```

Comment:
I am unsure what would be the `ss.getSource.getPath` in case of #13352 - e.g. what would be printed instead of
```
Error at 9-22 (e.g. `Visualization`) of 7b3fb3c2-6397-44b1-9518-e7414de00a44 with text:
Standard.Visualization.Table.Visualization.prepare_visualization _ (1000) <| column4 * 3
```
and whether that'd be useful. My idea of a fix is
```suggestion
                      .getCharEndIndex()} in ${if (ss.getSource.getLineCount > 2) ss.getSource.getPath else ss.getSource.getCharacters} (e.g. `${ss
```
but it is just a clueless fix without trying to understand the difference between visualization [with/without method pointer](https://github.com/enso-org/enso/blob/develop/docs/language-server/protocol-language-server.md#visualizationconfiguration). CCing @4e6

Reply:
> the expression was string (print text) or MethodPointer (print path).

I did check that and I'm afraid that logic is wrong. In all the cases that we've seen it's the former so we would end up in the same spot.

As I said, this has been a rather annoying regression and I'm partially to blame as I approved #13352. Feel free to readjust with any follow up PR.